### DISCUSSION TITLE
Cannot upgrade to PRO

### DISCUSSION THREAD

**Discussion by thoughtassassin** [vercel/community#6746] [Account Issue]
### Question

I exceeded the bandwidth for Blob storage on my hobby account. I tried to upgrade to PRO via the button that prompts you to upgrade to PRO. It took my credit card info but it did not upgrade the plan to PRO. I also see nothing in the settings to upgrade the project to PRO.

**Comment by vvo** [comment]
Hey @thoughtassassin sorry it took us so long to respond here. Did you manage to solve this? If not, can you reach out to https://vercel.com/help? As upgrade issues will get priority and we can help you from there. Thanks!
